projects/plugins/jetpack/extensions/store/wordpress-com/reducer.ts

```diff
@@ -16,7 +16,7 @@ const INITIAL_STATE: PlanStateProps = {
 			hasFeature: false,
 			isOverLimit: false,
 			requestsCount: 0,
-			requestsLimit: 0,
+			requestsLimit: 1000,
```

Comment:
I think we can settle for 20 on this one as it should be the free default value?

Reply:
or zero if we want to be restrictive with our defaults

Reply:
yes. Let me update the code